hello_world.py

```diff
@@ -1 +1,34 @@
-print("hello world")
+# importing random to make use of it for tuples
+import random
+
+print("Hello World")
+
+# Closes Issue: #9
+# Asks the user to enter their name and displays it to them
+userName = input("Enter your name: ")
```

Comment:
It seems like python community prefer using underscore syntax for variable names instead of camel cases :) 

Such as `user_name`!